GitUI/CommandsDialogs/FormFormatPatch.cs

```diff
@@ -243,6 +268,10 @@ private bool SendMail(string dir)
                 MessageBox.Show(this, ex.Message, TranslatedStrings.Error, MessageBoxButtons.OK, MessageBoxIcon.Error);
                 return false;
             }
+            finally
+            {
+                attachmentStreams.ForEach(x => x.Dispose());
```

Comment:
Do not introduce premature code pessimisation.
This works faster and has the same amount of characters.

```suggestion
                foreach (var x in attachmentStreams) x.Dispose();
```